content/docs/lists-and-keys.md

```diff
@@ -94,11 +94,11 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/jrXYRR?editors=0011)
+[**Experimente no CodePen**](https://codepen.io/gaearon/pen/jrXYRR?editors=0011)
 
-## Keys {#keys}
+## Chaves {#keys}
 
-Keys help React identify which items have changed, are added, or are removed. Keys should be given to the elements inside the array to give the elements a stable identity:
+As chaves ajudam a React identificar quais itens sofreram alterações, adicionadas ou removidas. As chaves devem ser atribuídas aos elementos dentro do array para dar uma identidade estável aos elementos:
```

Comment:
```suggestion
As Chaves ajudam React a identificar quais itens sofreram alterações, adicionadas ou removidas. As Chaves devem ser atribuídas aos elementos dentro do array para dar uma identidade estável aos elementos:
```

Reply:
I think in this case we should keep Chaves in lowercase, they only function as a common noun -- they don't have a special meaning in this context.